chemprop/cli/train.py

```diff
@@ -864,7 +864,7 @@ def train_model(
         trainer.fit(model, train_loader, val_loader)
 
         if test_loader is not None:
-            predss = trainer.predict(model, test_loader)
+            predss = trainer.predict(test_loader)
```

Comment:
To clarify, does this line mean that it will be tested on the best model?

Reply:
I ran some tests and can confirm that if the trainer isn't given a model, it will use the best validation score checkpoint model from the previous `trainer.fit` (assuming that they occur in the same function call as they do here).